website/docs/components/accordion/index.js

```diff
@@ -7,11 +7,32 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
+const STORAGE_KEY = 'website:accordion:item-2-state';
 export default class Index extends Component {
-  @tracked state = 'close';
+  @tracked accordionState = 'close';
+  @tracked itemState;
+
+  // Store Item state in session storage to persist across page reloads
+  constructor(owner, args) {
+    super(owner, args);
+
+    if (typeof window !== 'undefined' && typeof sessionStorage !== 'undefined') {
+      const saved = sessionStorage.getItem(STORAGE_KEY);
+        this.itemState = saved ?? 'open';
+      }
+  }
 
   @action
-  toggleState() {
-    this.state = this.state === 'open' ? 'close' : 'open';
+  toggleAccordionState() {
+    this.accordionState = this.accordionState === 'open' ? 'close' : 'open';
+  }
+
+  @action
+  onItemToggle() {
+    this.itemState = this.itemState === 'open' ? 'close' : 'open';
+     if (typeof window !== 'undefined' && typeof sessionStorage !== 'undefined') {
+      sessionStorage.setItem(STORAGE_KEY, this.itemState);
+    }
   }
```

Comment:
[suggestion] The error that you see in the console is related to fastboot rendering the page server-side; we can use a slight different approach to it, without checking the existence of `window` and `sessionStorage` which are always defined in a browser context:

```suggestion
import { tracked } from '@glimmer/tracking';
import { service } from '@ember/service';

const STORAGE_KEY = 'website:accordion:item-2-state';
export default class Index extends Component {
  @tracked accordionState = 'close';
  @tracked itemState;
  @service fastboot;

  // Store Item state in session storage to persist across page reloads
  constructor(owner, args) {
    super(owner, args);

    if (!this.fastboot.isFastBoot) {
      this.itemState = sessionStorage.getItem(STORAGE_KEY) ?? 'open';
    }
  }

@action
  toggleAccordionState() {
    this.accordionState = this.accordionState === 'open' ? 'close' : 'open';
  }

  @action
  onItemToggle() {
    this.itemState = this.itemState === 'open' ? 'close' : 'open';
    sessionStorage.setItem(STORAGE_KEY, this.itemState);
}
```